Tests/AfluentTests/ErrorsTests/TimeoutErrorTests.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  TimeoutErrorTests.swift
+//  Afluent
+//
+//  Created by Dalton Alexandre on 4/5/25.
+//
+
+
+import Testing
+@testable import Afluent
```

Comment:
We might consider not doing a `@testable import` and just `import Afluent` here, because all of what these tests assert should be true for public consumers of Afluent